apps/client/src/app/pages/voyage-tracker/voyage-tracker/vessel-row/vessel-build-column/vessel-build-column.component.html

```diff
@@ -3,7 +3,7 @@
        [nzPopoverTitle]="name"
        [nzPopoverContent]="vesselBuildDetails"
   >
-    {{ vesselBuild?.abbreviation }}
+    <div [ngClass]="conditionClass()" class="condition">{{ vesselBuild?.abbreviation }} </div>
```

Comment:
Calling a function in Angular template binding is a bad practice because the change detector will have to call it everytime it "travels" here to see if previous value is == new value, in order to know if yes or not, the binding should be updated.

I'll put a code suggestion in the TS part of it to explain how to implement this differently.